src/App.jsx

```diff
@@ -1,54 +1,22 @@
+/* eslint-disable arrow-parens */
 import './App.scss';
 
 import todosFromServer from './api/todos.json';
 import usersFromServer from './api/users.json';
+import { TodoList } from './components/TodoList';
 
 function getUserById(userId) {
-  return usersFromServer.find(user => user.id === userId)
-      || null;
+  return usersFromServer.find((user) => user.id === userId) || null;
 }
 
-export const todos = todosFromServer.map(todo => ({
+export const todos = todosFromServer.map((todo) => ({
```

Comment:
you wrap todo `param` with round braces but omit this in [todo](https://github.com/mate-academy/react_static-list-of-todos-js/pull/15/files#diff-0c1e77ea1bd7680b7677e171e3185d68432fa1c22fc12ef2c6a5906e72dfb74dR7) case - please be consistent and do it some one way